virttest/vt_agent/app/args.py

```
@@ -0,0 +1,42 @@
+def init_arguments():
+    """
+    Initialize the arguments from the command line.
+
+    :return: The populated namespace of arguments.
+    :rtype: argparse.Namespace
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--host",
+        action="store",
+        default="0.0.0.0",
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)